SPECS/kernel-mshv/config

```diff
@@ -1,19 +1,20 @@
 #
 # Automatically generated file; DO NOT EDIT.
```

Comment:
Was the existing config in main the fine-tuned version for kata? I can do a more careful comparison if so.

Reply:
Yes. Do be aware that https://github.com/microsoft/CBL-Mariner/pull/4691 is inbound and has the latest official kernel-mshv